rpc-server.go

```diff
@@ -111,22 +130,50 @@ func (s *storageServer) RenameFileHandler(arg *RenameFileArgs, reply *GenericRep
 }
 
 // Initialize new storage rpc.
-func newRPCServer(exportPath string) (*storageServer, error) {
+func newRPCServer(serverConfig serverCmdConfig) (servers []*storageServer, err error) {
```

Comment:
Can you add 'disks, ignoreDisks' ? rather than 'serverConfig?' - also there is a struct which is 'serverConfig' which would conflict here.